tests/mounttest.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/sh
+#
+# Basic test to ensure zbrewmount works for both mount and unmount
+#
+
+function crtzfs {
+    dsname=$1
+	cmdout="${ZBREW_TMP}/cmd.out"
+	touch "${cmdout}"
+	
+	# define a VSAM ZFS dataset 
+	mvscmdauth --pgm=IDCAMS --sysprint="${cmdout}" --sysin=stdin <<zzz
+	  DEFINE CLUSTER(NAME(${dsname}) -
+	  LINEAR TRACKS(10 10) SHAREOPTIONS(3))
+zzz
+	rc=$?
+	if [ $rc -gt 0 ]; then
+		echo "Error creating ZFS Linear Cluster:  $rc" >&2
+		cat "${cmdout}" >&2
+		return $rc
+	fi
+	mvscmdauth --pgm=IOEAGFMT --args="-aggregate ${dsname} -compat" --sysprint="${cmdout}"
+	rc=$?
+	if [ $rc -gt 0 ]; then
+		echo "Error formatting ZFS: $rc" >&2
+		cat "${cmdout}" >&2
+		return $rc
+	fi
+	rm -f "${cmdout}"
+}
+
+function cleanup {
+    drm -f "${ZBREW_SRC_HLQ}ZHW110*.**"
+    drm -f "${ZBREW_TGT_HLQ}ZHW110*.**"
+    rm $err 2>/dev/null
+}
+
+function assert {
+    expected=$1
+    actual=$2
+    if ! [ $expected -eq $actual ]; then
+	    echo "$(cat $err)"
+    fi
+}
+
+#main
+. zbrewsetenv
+
+export ZBREW_SRC_HLQ=ZBREWTS.
+export ZBREW_SRC_ZFSROOT=/zbrew/mts/
+export ZBREW_TGT_HLQ=ZBREWTT.
+export ZBREW_TGT_ZFSROOT=/zbrew/mtt/
+
+err="${ZBREW_TMP}/err.out"
+cleanup
+
+crtzfs ${ZBREW_SRC_HLQ}ZHW110.SZHWHFS
```

Comment:
i would recommend picking something other than ZHW110 so you don't have to worry about it colliding with the hello world package.